docs(guide): run ML on another machine

fabiannagel opened the pull request:
Based on [this discussion](https://github.com/immich-app/immich/discussions/2852#discussioncomment-6223779) from the 1.62.0 release.

fabiannagel committed:
Add documentation for hosting the ML container on a different machine

bo0tzz commented:
Awesome, thank you! To avoid filling the install guide too much, I think it might be better to make it a separate page under the 'Guides' section.

fyfrey commented:
> Awesome, thank you! To avoid filling the install guide too much, I think it might be better to make it a separate page under the 'Guides' section.

Agreed! It would also be good to discuss in the guide what happens if the other machine is not available (e.g. your laptop is powered off)

bo0tzz commented on docs/docs/install/docker-compose.md:
```diff
@@ -154,6 +154,40 @@ IMMICH_MACHINE_LEARNING_URL=http://immich-machine-learning:3003
+To alleviate [performance issues on low-memory systems](/docs/FAQ.md#why-is-immich-slow-on-low-memory-systems-like-the-raspberry-pi) like the Raspberry Pi, you may also host Immich's machine-learning container on a more powerful system (e.g. your laptop or desktop computer):
+
+- Set `IMMICH_MACHINE_LEARNING_URL` to point to the designated ML system, e.g. `http://workstation:3003`
+- Copy your `.env` file and the following `docker-compose.yml` to the ML system:
```
The .env file isn't actually necessary at all for the separate ML container.

alextran1502 commented:
Hello, can you help move the info to the suggested section?

alextran1502 commented:
Hello, any update on this PR?

fabiannagel committed:
Revert "Add documentation for hosting the ML container on a different machine"

This reverts commit 11e635eb57b25fc40c71edb72d5379e159724ec7.

fabiannagel committed:
Moved to Guides section and removed .env file reference

fabiannagel commented:
Sorry for the delay guys, I was quite busy the last weeks. Moved to the "Guides" section, hope the title is not too long. Also removed the `.env` file reference. Also added a comment about potential issues due to version mismatches.

jrasm91 commented:
Run npm run format:fix

jrasm91 reviewed (approved):
Looks good. Thank you.

If the container is offline, the app will still try to connect and there will just be a bunch of errors in the server logs I think. We can improve this behavior in the future.

alextran1502 committed:
format fix
